src/backends/emitutil.ts

```diff
@@ -23,6 +23,19 @@ export function progsym(progid: number) {
   return "q" + progid;
 }
 
+// Get an index variable name for a for loop,
+// (i.e., i in for (int i = 0; i < 5 i++) {stmts}).
+export let condsym = (() => {
+  let count = 0;
+  return () => {
+    return "c" + count++;
+  };
+}) ();
```

Comment:
Just for symmetry with the other symbol-generating utilities, what do you think about having this one take the ID of the AST node (here, the `WhileNode`)? That way, it will at least be possible to associate the variable name with something in the AST for debugging.

(I really need to get around to making the variable names more intelligible…)